app/src/main/java/com/android/partagix/model/ItemViewModel.kt

```diff
@@ -62,8 +65,14 @@ class ItemViewModel(
 
   /** Save the item with the current UI state in the database */
   fun save(new: Item) {
+    var category = new.category
+    if (category.name ==
+        "Category") { // todo : change so the when the users puts no category, the category is null
```

Comment:
```suggestion
        "Category") {
```
Category is the name of the field by default it will never be modified, user won't be able to change those categories (we will set a number of categories ourselves). Users will be able to create groups but not to modified categories